sweddit_combined/history/countries/CGS - Changsheng.txt

```diff
@@ -1,5 +1,4 @@
 government = monarchy
-add_government_reform = chinese_warlord
```

Comment:
Should changsheng (and similar tags) be left without a default reform instead of having confucian_state_reform to start with?

Reply:
The ones without reforms don't have cores in the beginning of the game. Only way for them to spawn is through rebels. If they still manage to appear they'll be able to pick whatever reform they want.